src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj

```diff
@@ -4,7 +4,7 @@
     <RootNamespace>Microsoft.Extensions.FileProviders</RootNamespace>
     <Description>File provider for files in embedded resources for Microsoft.Extensions.FileProviders.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
```

Comment:
To confirm: Arcade finds well-named .nuspec files automatically?

Reply:
Oops, not sure why my editor did this. Unintentional change. I’ll re-review my PR to make sure there are no other mistakes.